modules/cluster-logging-log-store-status-viewing.adoc

```diff
@@ -217,3 +217,29 @@ status:
       status: 'True'
       type: InvalidMasters
 ----
+
+
+The following status message indicates that Elasticsearch storage does not support the change you tried to make.
+
+For example:
+[source,yaml]
+----
+status:
+  clusterHealth: green
+  conditions:
+    - lastTransitionTime: "2021-05-07T01:05:13Z"
+      message: Changing the storage structure for a custom resource is not supported
+      reason: StorageStructureChangeIgnored
+      status: 'True'
+      type: StorageStructureChangeIgnored
+----
+
+The `reason` and `type` fields specify the type of unsupported change.
+`StorageStructureChangeIgnored`:: Unsupported change between ephemeral and persistent storage structures
+`StorageClassNameChangeIgnored`:: Unsupported change to the storage class name
+`StorageSizeChangeIgnored`:: Unsupported change the storage size
+
+[NOTE]
+====
+Suppose you configure the ClusterLogging custom resource (CR) to switch from ephemeral to persistent storage. In that case, the OpenShift Elasticsearch Operator creates a persistent volume claim (PVC) but does not create a persistent volume (PV). To clear the StorageStructureChangeIgnored status, you must revert the change to the ClusterLogging CR and delete the persistent volume claim (PVC).
```

Comment:
Suppose you configure the ClusterLogging custom resource (CR) to switch from ephemeral to persistent storage.

An incomplete sentence? Not sure.. please check and do the needful. Somehow not sitting well is what I feel.

Reply:
Will this help?

. In that case, ---> ; in that case,

Reply:
persistent volume claim (PVC) --->PVC 

since this is a 2nd occurrence, we can keep it as PVC. I think there is a 1st occurrence above (L176) wherein I have told you to expand.

Reply:
Updated. Thanks.